src/Models/TicketAttachment.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace dnj\Ticket\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class TicketAttachment extends Model
```

Comment:
please use [casts](https://laravel.com/docs/9.x/eloquent-mutators) to automatically serialize/deserialize  `file`.